tests/braille-specs/nemeth.yaml

```diff
@@ -0,0 +1,157 @@
+# Test file for nemethdefs.cti
+
+# FIXME: This test should be changed so that it tests a top-level
+# table (i.e. a table that can be discovered based on metadata). For
+# instance, the table could be nemeth.ctb, and the test would document
+# the math symbols as well as the "semantic" characters used for the
+# linearization of MathML (see also
+# https://github.com/liblouis/liblouis/issues/815).
+
+# Copyright © 2023 Attila Hammer from IT Foundation for the Visually Impaired - Hungary <www.infoalap.hu>
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+display: |
+  include unicode.dis
+  display \x00A0 a
+table: |
+  include nemethdefs.cti
+  include en-chardefs.cti
+  letsign 56
```

Comment:
nemeth.ctb does not include `letsign` so I wanted to remove the `letsign` rule from this test too. But when I do that some tests fail. What does the Nemeth specification say about this?